frontend/packages/client/src/components/Proposal/HeaderNavigation.js

```diff
@@ -1,6 +1,6 @@
 import { JoinCommunityButton } from 'components';
 import { useMediaQuery } from 'hooks';
-import BackButton from './BackButton';
+import BackButton from '../BackButton';
```

Comment:
maybe export `BackButton` in `components/index` and then combine this with line 1